src/components/ui/calendar/calendar.component.tsx

```diff
@@ -68,6 +68,10 @@ export type CalendarElement<D = Date> = React.ReactElement<CalendarProps<D>>;
  * to render instead of default year cell.
  * Called with a date for this cell and styles provided by Eva.
  *
+ * @property {(D) => void} onVisibleDateChange - Called when
+ * dates in visible date selection component changed. You should only focus on
+ * `newDate`'s year and month, date is not useful.
+ *
```

Comment:
I believe this will be more clear :)
```suggestion
 * @property {(D) => void} onVisibleDateChange - Called when navigating to the previous or next month / year.
 *
```